incremental-font-transfer/src/font_patch.rs

```diff
@@ -127,14 +156,41 @@ impl IncrementalFontPatchBase for FontRef<'_> {
             return Err(PatchingError::NonIncrementalFont);
         }
 
-        patch.apply_to(self)
+        let first = patches.first().ok_or(PatchingError::InvalidPatch(
```

Comment:
I'd personally probably just make this a variant, `PatchingError::EmptyPatch` or something.